templates/simple-broker-template.yaml

```diff
@@ -298,6 +298,7 @@ objects:
         ssl_cert_key: /etc/tls/private/tls.key
         ssl_cert: /etc/tls/private/tls.crt
         auto_escalate: ${AUTO_ESCALATE}
+        force_delete: ${FORCE_DELETE}
```

Comment:
It will be set to `false` by default from the params. See https://github.com/PhilipGough/ansible-service-broker/blob/b9f72c717d8ae3fe3321059daa2ee8c587c2e570/templates/simple-broker-template.yaml#L445-L448